### PR TITLE
Akhila/17

### LINKED ISSUE #17: Define movements to the player
As a Player, I will add movement to the player, so that can move on screen

### PR DESCRIPTION
**Pull request by AkhilaChiluveru**
User story- Adding movement to the player

### CODE REVIEW

File: blankwindow.py [hunk 1]
```diff
@@ -9,33 +9,40 @@
...
+bg_surf=pygame.image.load('./graphics/background.png').convert_alpha()
 
-#x is constant as player moves slowly to the top.
-ship_y_pos=500  
+#movements
+#movement of ship
+ship_rect=ship_surf.get_rect(center=(window_width/2,window_height/2))  
 
-#importing background
-bg_surf=pygame.image.load('hit-get/graphics/background.png').convert_alpha()
```
Comment (vkarinje) [inline blankwindow.py]: Why did not you use convert_alpha() for importing background?

File: blankwindow.py [hunk 2]
```diff
@@ -9,33 +9,40 @@
...
 #importing images 
-ship_surf = pygame.image.load('hit-get/graphics/ship.png').convert_alpha()
+#importing ship
+ship_surf = pygame.image.load('./graphics/ship.png').convert_alpha()
+#importing background
+bg_surf=pygame.image.load('./graphics/background.png').convert_alpha()
 
-#x is constant as player moves slowly to the top.
-ship_y_pos=500  
+#movements
```
Comment (vkarinje) [inline blankwindow.py]: Remove comment

**Review by vkarinje — COMMENTED** [review]
I think it is better not to do so many commits for one user story

File: blankwindow.py (hunk 1 above)
Comment (AkhilaChiluveru) [reply blankwindow.py]: The reason is- convert simply reduces png to a gameable thing. With alpha, it makes the image background transparent. We need not use alpha to make the background image transparent.

**Comment by AkhilaChiluveru** [comment]
Will make the necessary changes.

**Pull request closed without merging**